packages/cmrjs/utils.js

```diff
@@ -35,139 +46,58 @@ function hostId(env) {
-    }
-    else if (env === 'SIT') {
-      url = 'https://testbed.echo.nasa.gov/echo-rest/tokens/';
-    }
-    else {
-      url = 'https://api-test.echo.nasa.gov/echo-rest/tokens/';
-    }
-    break;
-  case 'search':
-    url = `https://${host}/search/`;
-    break;
-  case 'validate':
-    url = `https://${host}/ingest/providers/${provider}/validate/`;
-    break;
-  case 'ingest':
-    url = `https://${host}/ingest/providers/${provider}/`;
-    break;
-  default:
-    url = null;
-  }
-
-  return url;
-}
-
-
 /**
  * Find the UMM version as a decimal string.
  * If a version cannot be found on the input object
  * version 1.4 is assumed and returned.
  *
+ * @deprecated
+ *
  * @param {Object} umm - UMM metadata object
  * @returns {string} UMM version for the given object
  */
 function ummVersion(umm) {
+  deprecate('@cumulus/cmrjs/utils#ummVersion', '1.11.1');
   return _get(umm, 'MetadataSpecification.Version', '1.4');
 }
 
 /**
  * Transform UMM version number to metadata format string.
  *
  * @param {string} versionNumber - UMM version string in decimal format (e.g. 1.4)
+ * @param {string} [ummFormat='json'] - [optional] UMM format, defaults to 'json'
  * @returns {string} UMM-G metadata format string (e.g. umm_json_v1_4)
  */
 function ummVersionToMetadataFormat(versionNumber, ummFormat = 'json') {
   return `umm_${ummFormat}_v${versionNumber.replace('.', '_')}`;
 }
 
-/**
- * Posts a given xml string to the validate endpoint of the CMR
- * and returns the results
- *
- * @param {string} type - service type
- * @param {string} xml - the xml document
- * @param {string} identifier - the document identifier
- * @param {string} provider - the CMR provider
- * @returns {Promise.<boolean>} returns true if the document is valid
- */
-async function validate(type, xml, identifier, provider) {
-  let result;
-  try {
-    result = await got.post(`${getUrl('validate', provider)}${type}/${identifier}`, {
-      body: xml,
-      headers: {
-        'Content-type': 'application/echo10+xml'
-      }
-    });
-
-    if (result.statusCode === 200) {
-      return true;
-    }
-  }
-  catch (e) {
-    result = e.response;
-  }
-
-  const parsed = await new Promise((resolve, reject) => {
-    xml2js.parseString(result.body, xmlParseOptions, (err, res) => {
-      if (err) reject(err);
-      resolve(res);
-    });
-  });
-
-  throw new ValidationError(
-    `Validation was not successful, CMR error message: ${JSON.stringify(parsed.errors.error)}`
-  );
-}
-
 /**
  * Posts a given xml string to the validate endpoint of CMR
  * and promises true of valid.
  *
+ * @deprecated
+ *
  * @param {string} ummMetadata - the UMM object
  * @param {string} identifier - the document identifier
  * @param {string} provider - the CMR provider
  * @returns {Promise<boolean>} returns true if the document is valid
  */
 async function validateUMMG(ummMetadata, identifier, provider) {
+  deprecate('@cumulus/cmrjs/utils#validateUMMG', '1.11.1');
```

Comment:
Are these deprecations "as of 1.11.1" simply from before we released v1.11.[2|3]?

Reply:
Ahh yes, well it was 1.11.1 when I started this. I'll change all of these to 1.11.3